examples/mongodb-example/routes.js

```diff
@@ -1,7 +1,9 @@
 const User = require('./user');
+const { health } = require('../../build/middlewares');
```

Comment:
```suggestion
const { middlewares: {health} } = require('../../build');
``` 
Just to check that index.ts works.